src/sentry/stacktraces/processing.py

```diff
@@ -323,10 +325,21 @@ def normalize_stacktraces_for_grouping(
     # otherwise stored in `function` to not make the payload larger
     # unnecessarily.
     with sentry_sdk.start_span(op=op, description="iterate_frames"):
+        stripped_querystring = False
         for frames in stacktrace_frames:
             for frame in frames:
                 _update_frame(frame, platform)
 
+                if platform == "javascript":
+                    parsed_filename = urlparse(frame.get("filename", ""))
```

Comment:
is there any possible filename that could cause this to throw?

Reply:
I guess technically it could raise a `ValueError`: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse

That said, I don't think the SDK would/could actually send the kind of corrupt values it would take to cause that. Can't hurt to wrap it in a try-catch anyway, though.

Reply:
👍🏼